logger.js

```diff
@@ -65,6 +65,16 @@ Logger.prototype.error = function error(err) {
             fullType: err.fullType
         }));
 
+    } else if (err.type === 'tchannel.request.timeout') {
```

Comment:
Yes, it does. Whether the timeout happens in the local TChannel client or in a remote TChannel client is random. If it is local, we do not get an error frame.